ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/ovn/OWNERS

```diff
@@ -0,0 +1,8 @@
+approvers:
+- jianlinliu
+- gpei
+- jinyunma
```

Comment:
BTW, this is for GCP platform so the owner is "jianli-wei" instead of "jinyunma".

Reply:
Thanks @lihongan for reminding that. Taken care of